core/arch/riscv/kernel/console.c

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Commit "riscv: kernel: add console driver for S-Mode":

`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`